packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -154,4 +154,17 @@ export function setItemInLocalStorage(key, item) {
  */
 export function regexEncode(value) {
   return value.replace(/[-[\]/{}()*+?.\\^$|]/g, '\\$&');
+}
+
+/**
+ * Replaces certain characters in date string returned from Coveo Search API for browser compatibility. 
```

Comment:
The header comment describes which manipulations are performed, but not the function purpose. I'd suggest something like that.

    Converts a date string from the Coveo Search API format to the ISO-8601 format.